features/networking/service.feature

```diff
@@ -154,3 +154,43 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-11645
+  Scenario: Create loadbalancer service
+    Given I have a project
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/e3e88a1b3706a7ede5c6cb7805fa72d3218a9998/networking/ping_for_pod_containerPort.json |
+    Then the step should succeed
+
+    # Create loadbalancer service
+    When I run the :create_service_loadbalancer client command with:
+      | name | hello-pod |
+      | tcp  | 5678:8080       |
+    Then the step should succeed
+
+    # Get the external ip of the loadbaclancer service
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get client command with:
```

Comment:
@huiran0826 this step can be optimized by using the object's method.  It is not currently supported but #725 adds the support.

Reply:
@pruan-rht  Thanks for creating function in #725, when I use this function in the case, it has some issues.  Could you help take a look and update it?  Thanks!
1.  Currently the function loadbalancer_ingress get the result like this "[{"hostname"=>"af0790eba5c3611ea80c8066de33d3c7-1555561003.us-east-2.elb.amazonaws.com"}]", however what I need here is just value of hostname: 
 af0790eba5c3611ea80c8066de33d3c7-1555561003.us-east-2.elb.amazonaws.com. 

so in the function,  rr.dig('status', 'loadBalancer', 'ingress')  needs to update to rr.dig('status', 'loadBalancer', 'ingress')[0].dig('hostname') 

2. Secondly, when create loadbalancer service, it takes some time to populate the hostname.  At first, it just got empty result, after some seconds, can get correct results.
   status:
        loadBalancer: {}
------------------------
status:
        loadBalancer:
          ingress:
          - hostname: a4f00ddce5c3a11eab459026870219ce-746038338.us-east-2.elb.amazonaws.com

For the second issue, if you can add the check in the function, that will be better, if not, I can add it in the case, something like below.
And the expression should be true> cb.service_hostname.to_s.strip.length != 0

Please let me know what do you think, thanks!

Reply:
Updated some codes because there is some difference between gcp and aws, current codes work well.

Reply:
Could you try
And evaluation of `service("xxx").loadbalancer_ingress.first.dig("hostname")` is stored in the :yyy clipboard

Reply:
In aws, it is "hostname", but in gcp, it uses "ip",  and the current codes work well.
loadBalancer:
ingress:
- hostname: 

loadBalancer:
ingress:
- ip:

Reply:
@liangxia , Any other comments?  Can you help merge?